js/ui/grid_core/ui.grid_core.grid_view.js

```diff
@@ -618,17 +618,24 @@ var GridView = modules.View.inherit({
         });
     },
 
+    _getTableRoleName: function() {
+        return "grid";
+    },
+
     render: function($rootElement) {
         var that = this,
             isFirstRender = !that._groupElement,
             $groupElement = that._groupElement || $("<div>").addClass(that.getWidgetContainerClass());
 
         $groupElement.addClass(GRIDBASE_CONTAINER_CLASS);
         $groupElement.toggleClass(that.addWidgetPrefix(BORDERS_CLASS), !!that.option("showBorders"));
+
+        that.component.setAria({ "role": "presentation" }, $rootElement);
+
         that.component.setAria({
-            "role": "application",
+            "role": this._getTableRoleName(),
             "label": messageLocalization.format(that._getWidgetAriaLabel())
```

Comment:
Is this label needs if we add good role?

Reply:
I suppose this can be useful to understand what grid was selected..